src/Cake/Scripting/Roslyn/RoslynScriptSession.cs

```diff
@@ -96,10 +96,30 @@ public void Execute(Script script)
             var compilation = roslynScript.GetCompilation();
             var diagnostics = compilation.GetDiagnostics();
 
-            if (diagnostics.Any(d => d.Severity == DiagnosticSeverity.Error))
+            var errors = new List<Diagnostic>();
```

Comment:
@mholo65 am I missing something here, or are you never adding anything to `errors`?

Reply:
Oh, minor issue happened during refactoring. Nice catch!